RestSharp/Authenticators/HttpBasicAuthenticator.cs

```diff
@@ -44,9 +44,7 @@ public void Authenticate(IRestClient client, IRestRequest request)
 
             // only add the Authorization parameter if it hasn't been added by a previous Execute
             if (!request.Parameters.Any(p => p.Name.Equals("Authorization", StringComparison.OrdinalIgnoreCase)))
-            {
-                request.AddParameter("Authorization", this.authHeader, ParameterType.HttpHeader);
-            }
+                request.AddParameter("Authorization", authHeader, ParameterType.HttpHeader);
```

Comment:
Is it not a better coding convention to always have curly braces after an if?

Reply:
Why is it better? It just makes the code longer.

Reply:
It forces that:
```
if(true)
     doThis()
     doThat()
```
will never be interpreted as:
```
if(true)
{
     doThis()
     doThat()
}
```